tests/test_modeling_common.py

```diff
@@ -577,19 +577,46 @@ def test_training_gradient_checkpointing(self):
                 continue
             model = model_class(config)
             model.to(torch_device)
-            model.gradient_checkpointing_enable()
+            model.gradient_checkpointing_enable(gradient_checkpointing_kwargs={"use_reentrant": True})
```

Comment:
Nice! It's good to check explicity for the two `use_reentrant` values. We should still keep a test which checks the default behaviour though without kwargs (unless kwargs are enforced) i.e.: 

```model.gradient_checkpointing_enable()```

Reply:
Makes sense yes! Will modify accrondingly!